lib/utils/logger.js

```diff
@@ -0,0 +1,66 @@
+/* eslint import/no-dynamic-require: 0 */
+const os = require('os');
+const _ = require('lodash');
+const { check } = require('./environment-check');
+const type = require('./types');
+
+const {
+  createLogger,
+  format,
+  transports,
+} = require('winston');
+
+const {
+  combine,
+  timestamp,
+  printf,
+} = format;
+
+const logDirectory = `${os.homedir()}/.autolabjs`;
+
+check('LOGGERCONFIG');
+
+const config = require(process.env.LOGGERCONFIG);
+
+const logBlacklist = ['log', 'password'];
+
+const logFormat = printf((info) => {
+  if (info.module) {
+    return `${info.timestamp} [${info.level}] ${info.module.toUpperCase()}: ${info.message}`;
+  }
+  return `${info.timestamp} [${info.level}] MAIN: ${info.message}`;
+});
+
+// TODO: Improve structure logging
```

Comment:
really requires an alternative way of writing this code block.

Reply:
I wasn't very sure of how to go about modifying the structure. That's why I took the naive way and added a TODO.